src/TransformationFrontendAction.cpp

```diff
@@ -43,7 +48,16 @@ unique_ptr<ASTConsumer> TransformationFrontendAction::CreateASTConsumer(
 
 void TransformationFrontendAction::EndSourceFileAction() {
     SourceManager &SM = rewriter.getSourceMgr();
-    // Send rewritten buffer to std::out.
-    // TODO(write to file): change llvm::outs() to file specified by output_path
-    rewriter.getEditBuffer(SM.getMainFileID()).write(llvm::outs());
+    fs::path current_file_path(this->getCurrentFile().str());
+    fs::path transformations_path = fs::path(output_path) / current_file_path.stem();
+    size_t cur_transform_index = (size_t)distance(fs::directory_iterator{transformations_path},
+                                                  fs::directory_iterator{});
```

Comment:
Don't forget about description file. I suggest to you `count_if` with filtering cpp files.